.packit.yaml

```
@@ -0,0 +1,39 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-foreman-tasks.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-foreman-tasks.spec
+    - foreman-tasks.logrotate
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: foreman-tasks
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-foreman-tasks
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec -O rubygem-foreman-tasks.spec"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-foreman-tasks/foreman-tasks.logrotate -O foreman-tasks.logrotate"
+  get-current-version: "ruby -I lib/ -r foreman_tasks/version -e 'puts ForemanTasks::VERSION'"
+  create-archive:
+    - gem build foreman-tasks.gemspec
+    - bash -c "ls -1t ./foreman-tasks-*.gem | head -n 1"
```

gem build does output the filename so you can consider:
```suggestion
    - gem build foreman-tasks.gemspec | awk '/File: / { print $2 }'
```

But it may be better to explicitly determine the output filename:
```suggestion
    - gem build foreman-tasks.gemspec --output foreman-tasks.gem
    - echo foreman-tasks.gem
```

---

Doesn't the spec require the gem to be named like `%{gem_name}-%{version}`?

---

no, packit mangles the `Source0` line based on the output of this action :)